### ISSUE TITLE
Tuple modules may be going away

### ISSUE THREAD

**Issue by henrik** [antonmi/espec#52]
Happened across this, could be good to know: https://github.com/elixir-lang/elixir/issues/3254

[Pavlov](https://github.com/sproutapp/pavlov) has a nice take on RSpec-like syntax in Elixir without making use of tuple modules.

**Comment by antonmi** [comment]
Now, this is only proposal for elixir 2.0.
So, I don't think we should delete this 'rspec' syntax until elixir supports it.
But Pavlov style is perfect too.
Let's think is it possible to implement both syntaxes?

**Comment by henrik** [comment]
Just wanted to let you know :) I guess if it is removed there will be a long period of deprecation first, so probably no hurry.

**Comment by antonmi** [comment]
Hey, Henrik.
Thanks again for the issue!
I've implemented a new expectation syntax (the third one!).
See my latest commit.
Now you can write spec in the following way:
```ruby
context "simple specs" do
    it do: expect 1 |> to eq 1
    it do: expect 1 |> to_not eq 2
    it do: expect 1 |> not_to eq 2

    context "with subject" do
      subject 1
      it do: is_expected |> to eq 1
      it do: is_expected |> to_not eq 2
      it do: is_expected |> not_to eq 2
    end
  end
```
I think this is even more cool than pavlov approach!)
Thank again for the challenge!

**Comment by henrik** [comment]
@antonmi Oh, nice work! That seems like a pleasant syntax. Having `to eq` instead of Pavlov's `to_eq` is more like RSpec and could be more flexible if you support custom matchers.

**Comment by antonmi** [comment]
I've released 0.8.5 with the new syntax
